doc/nrf/protocols/bt/bt_mesh/dfu_over_bt_mesh.rst

```diff
@@ -201,12 +201,28 @@ Automated metadata generation
 By enabling the ``SB_CONFIG_DFU_ZIP_BLUETOOTH_MESH_METADATA`` option in sysbuild, the metadata will be automatically parsed from the ``.elf`` and ``.config`` files.
 The parsed data is stored in the :file:`ble_mesh_metadata.json` file.
 The file is placed in the :file:`dfu_application.zip` archive in the build folder of the application.
-Additionally, the metadata string required by the ``mesh models dfu slot add`` command will be printed in the command line window when the application is built::
+Additionally, the metadata string and firmware ID required by the ``mesh models dfu slot add`` command is printed in the command-line window when the application is built::
 
   Bluetooth Mesh Composition metadata generated:
     Encoded metadata: 020000000000000094cf24017c26f3710100
+    Firmware ID: 59000200000000000000
```

Comment:
May be worth doing this if you can spare couple of extra minutes. It is rather worth showing example with following numbers to show endianness conversion with action.

    fwid.company_id = 0x0059;
    fwid.major = 0xAA;
    fwid.minor = 0xBB;
    fwid.revision = 0x1234;
    fwid.build_num = 0x89ABCDEF;
    
  Which will become "5900aabb3412efcdab89"

Reply:
Users shouldn't need to care about the endianness of the FWID_MCUBOOT_VERSION IMO. Besides, the "5900" company ID already shows the little-endianness.

Reply:
> Users shouldn't need to care about the endianness of the FWID_MCUBOOT_VERSION IMO

I'm still uncertain about this. The model spec (I reffered initially to Access layer, but model layer has the same requirement) requires all data to be encoded into little endianness (section 3.8.1). The DFU server in our stack simply picks what points to fwid and passes it over the mesh network:
https://github.com/zephyrproject-rtos/zephyr/blob/baced6d315f0c58cf4950a8c49cb7a49d18c208f/subsys/bluetooth/mesh/dfu_srv.c#L171-L172

This works fine with our stack because we pass the fwid "as is" (as it comes in the message). But if some other stack is used, it can convert fwid into system endianness, and we will have problems here.

Reply:
@PavelVPV this was in response to Omkars suggestion about demonstrating endianness (of the MCUBOOT FWID!) in the user guide sample output. This is already little endian in my current implementation. I think you're confusing it with our separate discussion on endianness for the CUSTOM fwid.